src/cmcd.js

```diff
@@ -0,0 +1,324 @@
+import { CmcdObjectType } from '@svta/common-media-library/cmcd/CmcdObjectType';
+import { CmcdStreamType } from '@svta/common-media-library/cmcd/CmcdStreamType';
+import { CmcdStreamingFormat } from '@svta/common-media-library/cmcd/CmcdStreamingFormat';
+
+export class Cmcd {
```

Comment:
The CML handles assignment of CMCD keys to the correct header, so there is no need to divide them into separate classes. This new `Cmcd` class is just a consolidation of the previous `cmcdKeys` classes.

Reply:
The class should have a different name than the plugin class itself.